recipes/mothur/1.42.1/build.sh

```diff
@@ -0,0 +1,8 @@
+# the flags the new gxx activate script sets breaks the build, so unset them here
```

Comment:
Do you think we should remove those old versions?

Reply:
Are you sure removal would be the best course? Not sure if that would mess with people that already have them implemented. I can apply the same fixes as this version and they 'should' compile as desired.

Reply:
Removing the recipe does not mean removing the package. Lets formulate it the other way around, keeping the subdir means people want to maintain this particular version. So it gets rebuild against new python versoin etc. Is that really needed for this particual version?

Reply:
Ah, I understand what you're saying now. In that case, yeah, let's remove them. Should I just delete all the old subdirs then?

Reply:
Yes :)

Reply:
Ok, it should be all set to go now. Thanks for your help!